src/main/groovy/org/mockito/release/notes/internal/DefaultImprovement.java

```diff
@@ -87,4 +87,40 @@ public String toJson() {
     public void toJson(Writer writable) throws IOException {
         writable.append(toJson());
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
And I added equals/hashCode implementation only to the "domain" classes which are mostly used with collections